testing/acceptance_test.py

```diff
@@ -1238,3 +1238,24 @@ def test_3():
     assert (
         result.stdout.str().count("async def functions are not natively supported") == 1
     )
+
+
+def test_issue4445_rewrite(testdir):
+    """#4445: Make sure the warning points to a reasonable location
+    See origin of _issue_warning_captured at: _pytest.assertion.rewrite.py:241
+    """
+    testdir.makepyfile(some_mod="")
+    testdir.makeconftest(
+        """
+            import some_mod
+            import pytest
+
+            pytest.register_assert_rewrite("some_mod")
+        """
+    )
+    result = testdir.runpytest()
+    # with stacklevel=5 the warning originates from register_assert_rewrite
+    # function in the created conftest.py
+    result.stdout.fnmatch_lines(
+        ["*conftest.py:4*", "*Module already imported*", "*some_mod*"]
+    )
```

Comment:
I'm not sure `acceptance_test` is the best place.
From what I know it is meant as a top-level test module, and this appears to be something more detaled.
What about `test_warnings.py`?
(just a minor nit, not clear to me either)

Reply:
Yeah i am also not really sure. I just thought about putting it in there, because the issue #4445 mentioned acceptance tests.... But as your suggestion sounds reasonable i will put it in `test_warnings.py` then because the other tests are already written and also living in the `acceptance_test.py` but are not committed yet.

Reply:
it should go to the assertion rewriting tests

we need a better name for the historic acceptance tests file , but thats a followup

Reply:
Well those are testing warnings, so perhaps we can have a `TestStackLevel` class in `test_warnings` to group all those?

Reply:
Sounds reasonable, i will prepare that!